scripts/deploy.sh

```diff
@@ -27,6 +27,12 @@ function run_ansible {
   openstack-ansible ${ANSIBLE_PARAMETERS} --forks ${FORKS} $@
 }
 
+apt-get install python-dev -y
+curl -s -L https://bootstrap.pypa.io/get-pip.py -o /opt/get-pip.py
+python2 /opt/get-pip.py 'pip===7.1.2' || python /opt/get-pip.py 'pip===7.1.2'
+pip install --force-reinstall 'ansible===1.9.4'
```

Comment:
This appears to be duplicating parts of bootstrap-ansible.sh. Is there any reason the ansible-galaxy line can't be move to after the bootstrap-ansible.sh line?